index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const jestConfig = require('./jest-config');
+
+module.exports = jestConfig;
```

Comment:
This file seems unnecessary. I suggest pass the code from `jest-config.js` to here since it's already exporting the function we want.

Since we're not using JEST Presets, we don't really need files with specific names.